### PR TITLE
Libre2 NFC re-write squash

### PR DESCRIPTION
**Pull request by paulplant**

### CODE REVIEW

**Comment by paulplant** [comment]
Wrong repo specified in PR. Ignore.

File: xdrip/Calibration/NoCalibrator.swift [hunk 1]
```diff
@@ -45,6 +45,9 @@ class NoCalibrator: Calibrator {
         
         findSlope(for: bgReading, last2Readings: &last3Readings)
         
+        // to make sure that very high or very low values are capped
+        updateCalculatedValue(for: bgReading)
```
Comment (m-a-v) [inline xdrip/Calibration/NoCalibrator.swift:49]: Not sure if it makes sense to clamp high readings.

Yes. These should be rare, but so you don't see any trend of values that are over 400, even though they would be measured by the Libre.

I know the problem from loop users. I think it is not the right approach, or at least should be possible not to clamp the read values via options.

Comment (paulplant) [reply xdrip/Calibration/NoCalibrator.swift:49]: This is more of a safeguard and is just implemented the same as for Dexcom calibrator IIRC.

The actual fix for the Libre 2 high values (which where >20, 000, not just 500) is due to derived algo parameters not being cleared and is in another commit:

https://github.com/JohanDegraeve/xdripswift/pull/404/commits/eb281fe2f77cf1eafae1315fe9fd6b660754ab3a

Although this PR is closed, feel free to keep checking Sascha 🙏

Comment (m-a-v) [reply xdrip/Calibration/NoCalibrator.swift:49]: Thanks. But if I understand it correctly, values above 400 will be clamped now. Correct?

If yes: The information whether the blood sugar continues to rise or comes down (steeply) could still be helpful if it is above 400. As I said, these are hopefully rare moments, but they can happen and then this information helps whether the blood sugar continues to rise or fall.

**Pull request closed without merging**